plugin_runner/plugin_runner.py

```diff
@@ -286,27 +267,33 @@ async def HandleEvent(
 
         # Don't log anything if a plugin handler didn't actually run.
         if relevant_plugins:
+            # Send the Django request_started signal
+            request_finished.send(sender=None)
```

Comment:
do the same here please
```suggestion
            request_finished.send(sender=self.__class__)
```